discord/types/gateway.py

```diff
@@ -570,10 +570,10 @@ class PartialUpdateChannel(TypedDict):
 
 
 class PassiveUpdateEvent(TypedDict):
-    guild_id: Snowflake
-    channels: List[PartialUpdateChannel]
-    voice_states: NotRequired[List[VoiceState]]
-    members: NotRequired[List[MemberWithUser]]
+    removed_voice_states: List[Snowflake]
+    updated_channels: List[PartialUpdateChannel]
+    members: List[MemberWithUser]
+    updated_voice_states: List[VoiceState]
```

Comment:
Minor nitpick, but can you keep the old data structure for posterity and make a separate typed dict for the new one?